pkg/installer/args.go

```diff
@@ -34,40 +37,37 @@ const (
 
 // Args represents DirectPV installation arguments.
 type Args struct {
-	image       string
-	auditWriter io.Writer
+	image string
 
 	// Optional arguments
-	Registry                 string
-	Org                      string
-	ImagePullSecrets         []string
-	NodeSelector             map[string]string
-	Tolerations              []corev1.Toleration
-	SeccompProfile           string
-	AppArmorProfile          string
-	Quiet                    bool
-	KubeVersion              *version.Version
-	Legacy                   bool
+	Registry         string
+	Org              string
+	ImagePullSecrets []string
+	NodeSelector     map[string]string
+	Tolerations      []corev1.Toleration
+	SeccompProfile   string
+	AppArmorProfile  string
+	Quiet            bool
+	KubeVersion      *version.Version
+	Legacy           bool
+	ObjectWriter     io.Writer
+	DryRun           bool
+	Declarative      bool
+	ObjectConverter  func(runtime.Object) ([]byte, error)
```

Comment:
can we call it as Marshaller/ObjectMarshaller?

Reply:
done